sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.2 (2021-1-21)
 
 - Added diagnostic contexts to async APIs including service namespace.
+- Increased dependency version of azure-core from 1.11.0 to 1.12.0
+- Increased dependency version of azure-identity from 1.2.1 to 1.2.2
```

Comment:
worth mentioning the features/fixes we are getting from these upgrades.

Reply:
where were these versions bumped? in which file?

Reply:
I can add a link to the relevant azure-core and azure-identity changelogs, sure